ipod-shuffle-4g.py

```diff
@@ -464,7 +480,7 @@ def __init__(self, parent):
     def set_master(self, tracks):
         # By default use "All Songs" builtin voiceover (dbid all zero)
         # Else generate alternative "All Songs" to fit the speaker voice of other playlists
-        if self.playlist_voiceover and (Text2Speech.valid_tts['pico2wave'] or Text2Speech.valid_tts['espeak']):
+        if self.playlist_voiceover and (Text2Speech.valid_tts['pico2wave'] or Text2Speech.valid_tts['espeak'] or Text2Speech.valid_tts['say']):
```

Comment:
I am wondering if this additional and check is even required at all. But looking at the source it turns out there is lots of other stuff that can be improved, unrelated to your changes. So just treat this comment as me thinking out loud.

Reply:
There's another fork out there that improves TTS that I think would be nice to bring in.

Reply:
Can you point me to that fork?

Reply:
https://github.com/thomas-hori/IPod-Shuffle-4g/commit/0a72c1102d25101749279f6071ebacdd7f1ca7eb